README.md

```diff
@@ -69,35 +69,25 @@ started
 
 #### Configure Zilla to behave as a `tcp` `echo` server in 2mins.
 
-First create a local `zilla.json` with the following contents.
-```json
-{
-    "name": "example",
-    "bindings":
-    {
-        "tcp_server0":
-        {
-            "type" : "tcp",
-            "kind": "server",
-            "options":
-            {
-                "host": "0.0.0.0",
-                "port": 12345
-            },
-            "exit": "echo_server0"
-        },
-        "echo_server0":
-        {
-            "type" : "echo",
-            "kind": "server"
-        }
-    }
-}
-
-```
-Then run Zilla again, this time mounting your local `zilla.json` as a docker volume file.
-```
-docker run -v `pwd`/zilla.json:/zilla.json ghcr.io/aklivity/zilla:latest start -v
+First create a local `zilla.yaml` with the following contents.
+```yaml
+---
+name: example
+bindings:
+  tcp_server0:
+    type: tcp
+    kind: server
+    options:
+      host: 0.0.0.0
+      port: 12345
+    exit: echo_server0
+  echo_server0:
+    type: echo
+    kind: server
+```
+Then run Zilla again, this time mounting your local `zilla.yaml` as a docker volume file.
+```
+docker run -v `pwd`/zilla.yaml:/zilla.yaml ghcr.io/aklivity/zilla:latest start -v
```

Comment:
The docker run command needs to be updated same as on [Zilla Studio](https://zilla-studio.aklivity.io/#/). 😄 
```
docker run -v `pwd`/zilla.yaml:/etc/zilla/zilla.yaml ghcr.io/aklivity/zilla:latest start -v
```